packages/terra-core-docs/src/terra-dev-site/doc/action-header/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,265 @@
+import { Badge } from 'terra-action-header/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
+import Whitespace from "../../common/layout-helpers/Whitespace";
+import ActionHeaderWithDecorativeIcon from "./example/ActionHeaderWithDecorativeIcon";
+import ActionHeaderWithInformativeIcon from "./example/ActionHeaderWithInformativeIcon";
+import WebAIMSemanticHeadingContentStructure from './references/WebAIMSemanticHeadingContentStructure';
+import ActionHeaderNonUniqueIcon from './example/ActionHeaderNonUniqueIcon';
+
+
+
+<Badge />
+
+# Accessibility Guide for Terra Action Header
+
+
+## Why is this important?
+
+> Terra Action Header is an essential component used to identify a section of content and provide controls to manage the respective content. If Terra Action Header lacks accessibility, it can prevent users from understanding content and necessary context to interact with the page.  
+>
+>Action Headers provide a visual and programmatic presentation of content structure to convey a meaningful purpose to the section. Icon-only buttons are commonly use to visually communicate an action. Therefore, the buttons must be intuitive and have a programmatic association with other words on the page to ensure a unique and descriptive accessible control name.This will allow speech input devices to utilize them. Screen reader users will also rely on the added programmatic context to understand what precisely a control may do. 
```

Comment:
```suggestion
> Action Headers provide a visual and programmatic presentation of content structure to convey a meaningful purpose to the section. Icon-only buttons are commonly use to visually communicate an action. Therefore, the buttons must be intuitive and have a programmatic association with other words on the page to ensure a unique and descriptive accessible control name.This will allow speech input devices to utilize them. Screen reader users will also rely on the added programmatic context to understand what precisely a control may do. 
```